modules/es/fsw/src/cfe_es_task.c

```diff
@@ -38,7 +38,7 @@
 #include "cfe_version.h"
 #include "target_config.h"
 #include "cfe_es_verify.h"
-
+#include "cfe_es_api_typedefs.h"
```

Comment:
Also not needed ... I think we can back out changes to this file entirely (keep the changeset small and simple).

Reply:
All done - changeset a lot smaller now ;-)